server/Game.js

```diff
@@ -4,15 +4,33 @@ import { PING_TIMEOUT, SELECT_WORD_TIMEOUT, TICK_TIMEOUT } from '../shared/const
 import protocol from '../shared/protocol';
 import createHistory from './History';
 
-const words = require(`../${process.env.WORDS_FILE}`);
+const words = [...new Set(require(`../${process.env.WORDS_FILE}`))].filter(Boolean);
+
+if(words.length < 3) {
+  throw new Error('Expect to receive a list containing more than 3 valid and unique words.');
+}
 
 const encodeWord = (word) => [...word].map((c) => c === ' ' ? c : '_').join('');
 
+const shuffleArray = (originalArray) => {
+  const shuffledArray = [...originalArray];
+  
+  for(let i = shuffledArray.length - 1; i > 0; i--) {
```

Comment:
nit: prefer `const` whenever possible, `i`, `randomIndex` and `refWordSwaped` can all be const :-)

Reply:
Yeah! That makes perfect sense.
I just leave `i` as let because we are reassigning value when `i--` 💜

Reply:
Ah right, missed that one, thanks!